src/libraries/System.Runtime.Extensions/tests/System/Random.cs

```diff
@@ -414,6 +414,12 @@ public void Xoshiro_AlgorithmBehavesAsExpected()
             Assert.NotNull(s2);
             Assert.NotNull(s3);
 
+            if (!BitConverter.IsLittleEndian)
+            {
+                // The remainder of the test expects specific result values that will
+                // only happen on little-endian platforms.
+                return;
+            }
```

Comment:
The stuff above this isn't meaningful to test.  We should just disable the whole test on big-endian.

Reply:
I agree. You can do that by adding the following attribute to the test:

```csharp
[ConditionalFact(typeof(BitConverter), nameof(BitConverter.IsLittleEndian))]
```

Reply:
@akoeplinger This doesn't appear to work for me, the test case still fails and I get:
`System.InvalidOperationException : An appropriate member 'IsLittleEndian' could not be found. The conditional method needs to be a static method or property on the type System.BitConverter or any ancestor, of any visibility, accepting zero arguments, and having a return type of Boolean.`

Reply:
We should update the attribute discoverers to allow fields as well.

Reply:
I've seen other tests use a local property, so I've tried this and it seems to work.  Patch updated accordingly.   Does this look OK?

Reply:
> We should update the attribute discoverers to allow fields as well.

https://github.com/dotnet/arcade/pull/7117